src/Microsoft.OData.Core/JsonLight/ODataJsonLightReaderNestedResourceInfo.cs

```diff
@@ -115,24 +171,18 @@ internal IEdmStructuralProperty StructuralProperty
         {
             get
             {
-                return this.nestedProperty as IEdmStructuralProperty;
+                return this.NestedProperty as IEdmStructuralProperty;
             }
         }
 
         /// <summary>
-        /// The Edm property for which the nested resource info will be reported.
+        /// Whether individual members of the collection should be streamed.
         /// </summary>
-        internal IEdmProperty NestedProperty
-        {
-            get
-            {
-                return this.nestedProperty;
-            }
-        }
+        internal bool StreamMembers { get; set; }
 
         /// <summary>
-        /// true if the nested resource info has a value.
-        /// </summary>
+                 /// true if the nested resource info has a value.
+                 /// </summary>

```

Comment:
revert

Reply:
I don't see a change on this line; what are you suggesting be reverted?

---
In reply to: [259079080](https://github.com/OData/odata.net/pull/1204#discussion_r259079080) [](ancestors = 259079080)